.github/workflows/size.yml

```diff
@@ -18,7 +18,7 @@ jobs:
 
       - name: Resize images
         run: |
-          docker run -d --rm --name imagemagick -v $(pwd)/linuxserver.io:/inn -e FOLDER='/inn/img/' -e WIDTH=192 -e HEIGHT=192 ghcr.io/linuxserver/baseimage-alpine:3.18
+          docker run -d --rm --name imagemagick -v $(pwd)/linuxserver.io:/inn -e FOLDER='/inn/img/' -e WIDTH=192 -e HEIGHT=192 ghcr.io/linuxserver/baseimage-alpine:3.19
```

Comment:
Might be worth just using `alpine:3` so we don't need to remember to bump it every 6 months